apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -44,7 +44,7 @@ defmodule OMG.Watcher.ExitProcessor.Core do
 
   use OMG.Utils.LoggerExt
 
-  @default_sla_margin 10
+  @default_sla_seconds 10
```

Comment:
Not sure should the value reflect the original value, which I think should be 10 block * 15s per block = 150?

Reply:
true, I think these are for tests. Fails when sla_seconds is >10

Reply:
if this is for tests then let's remove it. we should also remove the default setting below:
`sla_seconds \\ @default_sla_seconds`